### ISSUE TITLE
Make HTML accessible 

### ISSUE THREAD

**Issue by RaHughes** [kaylaewood/RefactorTractor#1]
There are very few accessible tags in the HTML currently. There are no alt text properties on any of the tags currently. Using a screen reader as it stands now would be difficult.
